source/channels/message-priority.rst

```diff
@@ -22,25 +22,24 @@ From Mattermost v7.7, you can add a message priority label to root messages to m
 Set message priority
 --------------------
 
-You can add one of three priority labels when sending a new message: Standard, Important, or Urgent. 
+You can add one of three priority labels when sending a new message: Standard, Important, or Urgent. When you send a priority message, the priority label displays next to your name in the channel and the Threads view when others reply to the thread. 
 
 To set the priority of a new message:
 
 1. Select the **Message Priority** |priority-icon| icon in the message formatting toolbar. 
 2. Select the priority for the message. Messages have a standard priority by default.
 
-When you send the message, the priority label displays next to your name in the channel and the Threads view when others reply to the thread. 
-
 Request acknowledgements
 ------------------------
 
 .. include:: ../_static/badges/ent-pro-only.rst
   :start-after: :nosearch:
 
-`Mattermost Professional or Enterprise <https://mattermost.com/pricing>`__ customers can additionally request that recipients actively acknowledge the message to track that messages have been seen and actioned. By default, marking a message as Urgent priority automatically requests an acknowledgement. The ability to request acknowledgements and acknowledge messages is available using Mattermost in a web browser or the desktop app with mobile support coming in a future release. 
+`Mattermost Professional or Enterprise <https://mattermost.com/pricing>`__ customers can additionally request that recipients actively acknowledge the message to track that messages have been seen and actioned. By default, marking a message as Urgent priority automatically requests an acknowledgement. 
 
 When you request acknowlegement of a message, an **Acknowledge** |acknowledge-button| button is added below the sent message. You can mark message as acknowledged by selecting the button, and you can hover over the **Acknowledged** |acknowledged-icon| icon to review who has acknowledged the message.
 
 .. tip::
 
-    After acknowledging a message, you have up to five minutes to change your mind. Select the **Acknowledged** |acknowledged-icon| button again to remove your name from the list of acknowledged users.
\ No newline at end of file
+    - On mobile, you'll be notified every five minutes until you acknowledge or reply to the message.
```

Comment:
@larkox - Does this functionality require the user to have notifications enabled (at all or in some specific way)? I ask because I wasn't able to see this repeated notification functionality working (but I have notifications disabled via mobile, and suspect that's why).

Reply:
Not really sure. I haven't been much involved with this feature. Maybe @Willyfrog can add more context, so we can answer such question.